worlds/sm/variaRandomizer/rom/rom.py

```diff
@@ -89,9 +89,12 @@ def fillToNextBank(self):
 
 class FakeROM(ROM):
     # to have the same code for real ROM and the webservice
-    def __init__(self, data={}):
+    def __init__(self, data=None):
         super(FakeROM, self).__init__()
-        self.data = data
+        if data is None:
+            self.data = {}
+        else:
+            self.data = data
```

Comment:
Nitpicking: You could get this done without the else just by changing the parameter instead of setting the field directly.

```suggestion
        if data is None:
            data = {}
        self.data = data
```

Reply:
Performance may not matter here, but I write it like this because it is slightly more performant in the `data is None` case, skipping the assignment to `data` and skipping getting `data` from the locals.

Reply:
Sure I'm just nitpicking, feel free to dismiss my comment.